components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/bean/Application.java

```diff
@@ -21,57 +21,66 @@
 import java.io.Serializable;
 import java.util.List;
 
+/**
+ * This object contains the context related to OAuth application.
+ */
 public class Application implements Serializable {
 
     private static final long serialVersionUID = -4515815791420125411L;
 
-    private String client_name = null;
-    private String client_id = null;
-    private String client_secret = null;
-    private List<String> redirect_uris = null;
+    private String clientName = null;
+    private String clientId = null;
+    private String clientSecret = null;
+    private List<String> redirectUris = null;
+
+    public String getClientName() {
 
-    public String getClient_name() {
-        return client_name;
+        return clientName;
     }
 
-    public void setClient_name(String clientName) {
-        this.client_name = clientName;
+    public void setClientName(String clientName) {
```

Comment:
update the usages of the setters and getters

Reply:
fixed